docs/rdataretriever.rst

```diff
@@ -23,11 +23,19 @@ To install the development version, use ``devtools``
   # install.packages("devtools")
   library(devtools)
   install_github("ropensci/rdataretriever")
-
+  
 Note: The R package takes advantage of the Data Retriever's command line
 interface, which must be available in the path. This should occur automatically
 when following the installation instructions for the Retriever.
 
+:: 
+  library(rdataretriever)
+  # Set the Retriever's path 
+  rdataretriever::use_RetrieverPath('/home/ubuntu/.local/bin')
+
```

Comment:
Curious if this is the result of your retriever path. `/home/ubuntu/.local/bin`
or  it is `/home/ubuntu/..local/bin/retriever`